nephio-ansible-install/roles/tree/tasks/main.yaml

```diff
@@ -0,0 +1,7 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
```

Comment:
Do the above three lines need to be removed?

Reply:
I am discussing with some Nokia folks to remove this

Reply:
fixed https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/30345990bf7948269fe4954e827cb8740fb3984d

Reply:
Hello 
I succeeded to install entire environment by those ansible scripts.
I started to perform exercise from one-summit-22 with UPF 5g core deployment.  But when I open web ui on local browser there is no button "add organizational blueprint" in screen Organizational Blueprints